app/markets/ActivityTable.tsx

```diff
@@ -6,71 +11,69 @@ import {
   TableHeader,
   TableRow,
 } from '@/app/components/ui/Table';
-import { TransactionType, getMarketTransactions } from '@/queries/omen';
+import {
+  FixedProductMarketMaker,
+  FpmmTrade_OrderBy,
+  OrderDirection,
+  Scalars,
+  getMarketTrades,
+} from '@/queries/omen';
 import {
   formatDateTime,
   formatEtherWithFixedDecimals,
   getGnosisAddressExplorerLink,
   shortenAddress,
 } from '@/utils';
 import { Button, Icon, Tag, TagColorSchemeProp } from '@swapr/ui';
-import { useQuery } from '@tanstack/react-query';
-import { cx } from 'class-variance-authority';
-import { useState } from 'react';
 
-const getTagColorScheme = (transactionType: TransactionType): TagColorSchemeProp => {
-  switch (transactionType) {
-    case TransactionType.Buy:
-      return 'success';
-    case TransactionType.Sell:
-      return 'danger';
-    case TransactionType.Add:
-      return 'info';
-    case TransactionType.Remove:
-      return 'quaternary';
-    default:
-      return 'info';
-  }
+const getTagColorScheme = (
+  outcome: Scalars['String']['output'],
+  outcomes: NonNullable<FixedProductMarketMaker['outcomes']>
+): TagColorSchemeProp => {
+  if (outcome === outcomes[0]) return 'success';
+  else if (outcome === outcomes[1]) return 'danger';
```

Comment:
let's use outcome Index instead of outcome object here.

Reply:
Let's have an object instead of a function. Outcomes will be between 0 and 1 so we can have an object with `1: "success", ...`.

Reply:
Ah, you're right, makes much more sense, implementing this one now!